docs/src/pages/en/themes/responsive-themes.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Create a responsive theme
+description: Learn how to create a responsive theme
+layout: ../../../layouts/MainLayout.astro
+---
+
+There's a few ways to implement layouts that change based on context, one is
+explained in [Responsive controls](../responsive-controls) which is based on
```

Comment:
I think it's worth adding a sentence to the intro of the responsive controls docs that says something like "For more advanced use cases, check out Themes"

Reply:
I'll keep this for an iteration because I'm seeing we're also missing a section on the new Media Chrome breakpoints themselves. We should explain that first before directing them to themes